### PR TITLE
Aerodoc

### PR DESCRIPTION
**Pull request by danielpassos**
This PR:

* Import AeroDoc from [aerogear-aerodoc-android](https://github.com/aerogear/aerogear-aerodoc-android)
* Update AeroDoc to use AeroGear Android 2.0
* Replace Maven project by Gradle project

Related Jiras:
* [AGPUSH-1012](https://issues.jboss.org/browse/AGPUSH-1012)
* [AGDROID-362](https://issues.jboss.org/browse/AGDROID-362)

### CODE REVIEW

**Comment by danielpassos** [comment]
@sebastienblanc @matzew Could you review it?

File: AeroDoc/app/src/androidTest/java/org/jboss/aerogear/android/cookbook/aerodoc/ApplicationTest.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.jboss.aerogear.android.cookbook.aerodoc;
```
Comment (matzew) [inline AeroDoc/app/src/androidTest/java/org/jboss/aerogear/android/cookbook/aerodoc/ApplicationTest.java]: header ?

**Comment by matzew** [comment]
I have tested the application on my Galaxy SII - works perfect!

good job!

File: AeroDoc/app/src/androidTest/java/org/jboss/aerogear/android/cookbook/aerodoc/ApplicationTest.java (hunk 1 above)
Comment (danielpassos) [inline AeroDoc/app/src/androidTest/java/org/jboss/aerogear/android/cookbook/aerodoc/ApplicationTest.java]: Only because I want to see you happy :)

**Comment by matzew** [comment]
:+1:

**Pull request merged**